src/web/aws-cloudwatch/CloudWatch.jsx

```diff
@@ -0,0 +1,145 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
```

Comment:
I add the comment here, but my linter complains in different components about errors in them  :)